inductiva/tests/templating/test_template_engine.py

```diff
@@ -0,0 +1,129 @@
+"""Test the TemplateEngine mixin."""
+import os
+import pathlib
+import tempfile
+
+import pytest
+
+from inductiva import TemplateEngine
+
+ASSETS_DIR = pathlib.Path(__file__).parent / "assets"
+
+
+@pytest.fixture
+def fixture_templatr():
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        os.chdir(tmpdirname)
+        yield TemplateEngine(ASSETS_DIR)
+
+
+@pytest.fixture(name="templatr")
```

Comment:
this one (and the ones below) are not fixtures; they are using a fixture. They shouldn't have the decorator.